addons/hr_skills/static/src/fields/boolean_confirm.js

```diff
@@ -0,0 +1,29 @@
+/** @odoo-module */
+
+import { registry } from '@web/core/registry';
+import { BooleanToggleField, booleanToggleField } from "@web/views/fields/boolean_toggle/boolean_toggle_field";
+
+export class BooleanToggleSaveForm extends BooleanToggleField {
+    async onChange(value) {
+        await super.onChange(value);
+        // find the ListRenderer parent and get the ids list without the current record being toggled
+        let parent = this.__owl__.parent;
+        const record_id = parent.props.record.data.id;
```

Comment:
Wouldn't it be better/simpler to do this logic on the list view itself instead of having a field that will act on its siblings? :thinking:

Reply:
There would only be a JS class on the list view, and in the model's `save()` you can check if the `default_level` is set to true on a record and from there toggle all the others to off

Reply:
updated, managed to handle this in the write() on backend side.